app/assets/javascripts/lit/sources.js.coffee

```diff
@@ -0,0 +1,26 @@
+$ ->
+  sourceId = $('#source_id').attr('value')
+  apiKey = $('#api_key').attr('value')
+
+  updateFunc = ->
+    $.ajax "/lit/api/v1/sources/" + sourceId + "/sync_complete",
+      type: 'GET'
+      headers:
+        'Authorization': "Token token=\"#{apiKey}\""
+      success: (xml, textStatus, xhr) ->
+        if xhr.responseJSON.sync_complete
+          $('.loading').addClass('loaded').removeClass('loading')
+          clearInterval(interval)
+          location.reload()
+      statusCode:
+        404: ->
+          $('.loading').text('Could not update synchronization status, please try refreshing page')
+        401: ->
+          $('.loading').text('Invalid API key, please check source settings')
```

Comment:
@Bonias: 
I do see a point in moving it away from the API namespace since it is indeed not an operation relevant to a remote client consuming the API.
Regarding the possibility of placing `sync_complete` in `Lit::IncommingLocalizationsController`, I initially decided against it to keep an intuitive API shape - we're asking about the status of a `Source` and it seemed most logical to make `sync_complete` an action associated with the object. I believe that if we want to move it away from the API namespace, it could make more sense to move it to `Lit::SourcesController`.